pilot/pkg/model/context.go

```diff
@@ -93,6 +94,8 @@ func NewEnvironment() *Environment {
 	}
 }
 
+type Watcher = meshwatcher.WatcherCollection
```

Comment:
I'm guessing this is just to reduce the renaming?

Reply:
Yeah. I can split this stuff out as well if this PR is too huge to review